etc/org.opencastproject.assetmanager.aws.s3.AwsS3AssetStore.cfg

```diff
@@ -0,0 +1,17 @@
+# Configuration for AWS S3 Asset Manager Service
+
+# The AWS region. Default is us-east-1. Name must be a valid AWS region name like us-east-1.
+#org.opencastproject.assetmanager.aws.s3.region=
```

Comment:
Comment needs to be fixed. There's no default value.

Reply:
I am wondering if it would be best to have a "enabled=true/false" property because an error will show up in the log if this service is not being used: 
2018-09-07 13:36:17,470 | ERROR | (?:?) - [org.opencastproject.assetmanager.aws.s3.AwsS3AssetStore(212)] The activate method has thrown an exception
org.opencastproject.util.ConfigurationException: org.opencastproject.assetmanager.aws.s3.bucket is missing or invalid

Reply:
That's fair, done.